static/app/utils/replays/hooks/useReplayLayout.tsx

```diff
@@ -66,9 +69,17 @@ function useActiveReplayTab() {
       [defaultLayout, paramValue]
     ),
     setLayout: useCallback(
-      (value: string) =>
-        isLayout(value) ? setParamValue(value) : setParamValue(defaultLayout),
-      [defaultLayout, setParamValue]
+      (value: string) => {
+        const chosenLayout = isLayout(value) ? value : defaultLayout;
+
+        setParamValue(chosenLayout);
+        trackAdvancedAnalyticsEvent('replay-details.layout-changed', {
```

Comment:
Should we have a single "namespace"? e.g. `replay.details-viewed`, `replay.layout-changed` (details is kind of redundant here, but could include it to be consistent: `replay.details-layout-changed`).

Reply:
Good idea. It's the `replayAnalyticsEvents.tsx` file, so replay as a single prefix makes sense.